.travis.yml

```diff
@@ -1,26 +1,19 @@
-sudo: required
-
+dist: bionic
```

Comment:
The TravisCI default is xenial (ubuntu 16.04), and I'm confident there won't be any difference using the newer bionic (ubuntu 18.04). I figure it could be a good idea to keep a bit updated though as if we would use minikube to run tests etc, there may be a difference and a bit harder to get it working in my experience.